onnxruntime/python/onnxruntime_collect_build_info.py

```diff
@@ -0,0 +1,99 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import warnings
+import ctypes
+import sys
+
+
+def find_cudart_versions(build_env=False):
+    # ctypes.CDLL and ctypes.util.find_library load the latest installed library.
+    # it may not the the library that would be loaded by onnxruntime.
+    # for example, in an environment with Cuda 11.1 and subsequently
+    # conda cudatoolkit 10.2.89 installed. ctypes will find cudart 10.2. however,
+    # onnxruntime built with Cuda 11.1 will find and load cudart for Cuda 11.1.
+    # for the above reason, we need find all versions in the environment and
+    # only give warnings if the expected cuda version is not found.
+    # in onnxruntime build environment, we expected only one Cuda version.
+    if 'linux' not in sys.platform:
+        warnings.warn('find_cudart_versions only works on Linux')
+        return None
+
+    cudart_possible_versions = {None}
+    if not build_env:
+        # if not in a build environment, there may be more than one installed cudart.
+        cudart_possible_versions.update({
+            '11.3',
+            '11.2',
+            '11.1',
+            '11.0',
+            '10.2',
+            '10.1',
+            '10.0'})
```

Comment:
Do we need th explicitly list the versions? This might increase maintenance cost in future for more cuda versions.

Reply:
I have the list removed so that we only to find cudart libraries like cudart.so and audart.cuda_version.